src/app/helpers/page/pageRouting.ts

```diff
@@ -17,8 +18,7 @@ export function GetRouteConfigForPage(
 ) {
   const page = getPageInfo(component);
 
-  // tslint:disable-next-line: triple-equals
-  if (!page || page.route.fullRoute == null) {
+  if (!page || isUninitialized(page.route.fullRoute)) {
```

Comment:
inline this, use typescript Elvis operator

further, why is the extra condition needed? 

it kinda smells like another root issue that we're bypassing here... page.route should always be defined no?